src/EncompassRest/Loans/Enums/FnmCommunityLendingProductName.cs

```diff
@@ -0,0 +1,16 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum FnmCommunityLendingProductName
+    {
```

Comment:
numeric values to match string values (eg. n04 = 4)

Reply:
In case you didn't see I've done as you suggested.